docs/source/model-registry.rst

```diff
@@ -600,6 +605,72 @@ save, log, register, and load from the Model Registry and score.
     <Yay!! Another good phone interview. I nailed it!!> -- {'neg': 0.0, 'neu': 0.446, 'pos': 0.554, 'compound': 0.816}
     <This is INSANE! I can't believe it. How could you do such a horrible thing?> -- {'neg': 0.357, 'neu': 0.643, 'pos': 0.0, 'compound': -0.8034}
 
+Using Registered Model Aliases
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. note::
+    Registered model aliases are experimental and subject to change.
```

Comment:
Did we forget to annotate the actual `set_registered_model_alias` API (https://mlflow.org/docs/latest/python_api/mlflow.client.html?highlight=set_registered_model_alias#mlflow.client.MlflowClient.set_registered_model_alias) as experimental when we first released it? I don't think we can retroactively mark as experimental

Reply:
Yeah 🤦‍♂️ I didn't realize that the API docs were auto-generated from the function comments. Do you think it makes more sense to remove `experimental` from here or go add it to the API docs?

Reply:
I think we can just remove it from here, we don't expect to drop/deprecate the feature anyways or make breaking API changes